test/test_hex_binary.py

```diff
@@ -19,29 +19,29 @@ def _test_integer(self, i):
 
         l = Literal(hex_i, datatype=XSD.hexBinary)
         bin_i = l.toPython()
-        self.assertEquals(int(binascii.hexlify(bin_i), 16), i)
+        self.assertEqual(int(binascii.hexlify(bin_i), 16), i)
```

Comment:
Just for other reviewers. Some background is here: https://stackoverflow.com/a/931011/414075 and here: https://docs.python.org/3/library/unittest.html#deprecated-aliases